### DISCUSSION TITLE
Is it possible to get hwnd handle of the WindowHandle object ?

### DISCUSSION THREAD

**Discussion by deflexor** [rkarp/winapi-easy#6] [Q&A] [answered]
Original window ID is very useful in some  cases, for example taking screenshot of the window. But I examined this struct
and didnt find any way to get that handle.

**Comment by rkarp** [comment, marked as answer]
I did add a method `into_raw_handle` for this, which will give you a pointer. `winapi` models these handles as `*mut HWND__`, even though in this case it's just a `isize` under the hood. To get that here, you can just do something like `window_handle.into_raw_handle().as_ptr() as isize`.

Not very pretty, the new `windows` library makes the underlying type much more obvious, see: https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Foundation/struct.HWND.html

I may complete the switch to this new library at some point, and then the API in winapi-easy may change as well, especially `into_raw_handle`. It's all experimental still.
